score-server/src/main/resources/application.yml

```diff
@@ -82,6 +82,12 @@ auth:
     # E.g. TIMEOUT(n) = TIMEOUT(n-1) * MULTIPLIER
     multiplier: 2.0
 
+management:
+  endpoints:
+    web:
+      cors:
+        allowedOrigins: http://example:8080, http://localhost:8081, http://hello:8080
```

Comment:
can we leave only localhost origin by default? I guess others are just an example, also would be nice to add a comment to describe the value could contain multiple origins separated by commas.

Reply:
Done!